docker/test/integration/minifi/flow_serialization/Minifi_flow_yaml_serializer.py

```diff
@@ -92,21 +108,22 @@ def serialize(self, connectable, root=None, visited=None):
                     res['Connections'].append({
                         'name': str(uuid.uuid4()),
                         'source id': str(connectable.uuid),
-                        'source relationship name': conn_name,
                         'destination id': str(proc.uuid),
                         'drop empty': ("true" if proc.drop_empty_flowfiles else "false")
```

Comment:
Not related to this change, but do you know why `drop empty` is not included in the other (single-outgoing-connection) case?  That seems to be the only difference in the two blocks, without this, the duplication could be removed.

Reply:
Unfortunately, I don't know why is it not included there, but it seems to me as well that there shouldn't be a difference there. I added it in e64b1c0d1fe231936dcde263939fbd8dcfff4083